docs/wip/rendering.md

```diff
@@ -1,9 +1,31 @@
 # Types of rendering: SSG, SSR, CSR
 
+No mundo do React existem formas diferentes de renderizar páginas:
+
+### CSR (client-side rendering)
+
+É quando um arquivo HTML vazio é enviado para o navegador e, em seguida, o HTML é criado pelo Javascript. Isso ocorre porque o código do React precisa ser inicializado para renderizar os componentes na tela antes que ele possa cuspir HTML para o navegador analisar. Se você tiver muitos componentes carregando na tela, poderá ter problemas se quiser manter o tempo de interação (TTI) baixo. Esse cenário é onde o SSR frequentemente entra em jogo.
+
+### SSR (server side render)
+
+Como o React precisa inicializar em algum lugar, agora a renderização inicial acontece no servidor. Então, para cada solicitação que o usuário envia para o servidor, você ativa uma instância do React e manda para o navegador o HTML e CSS pronto.
+
+Dito isso, como você está contando com a funcionalidade do servidor para fazer essa renderização, é necessário ter uma configuração de servidor personalizada. Não há hospedagem CDN simples aqui - seu servidor precisa inicializar e renderizar a página de cada usuário mediante solicitação.
+
+### SSG (static site generator)
+
+Agora a renderização acontece em tempo de compilação, isso ocorre porque trazemos o servidor para nossa máquina e toda vez que compilamos nossa aplicação, a saída são arquivos estáticos que podemos servidor em CDN.
```

Comment:
```suggestion
## Static rendering

Static rendering generates a single HTML file for every page the user can access even before he requests it. That means that, differently from server-side rendering, which happens on-demand, static rendering happens once at build time.

Therefore, Static rendering brings the server to our local machines in a way that every time we compile our application, we get static files as our output in advance. These files can then be served on a CDN.

To handle Static rendering, we can work with tools such as [Gatsby](https://www.gatsbyjs.com/), a React-based static site generator.
```

Reply:
@carolinamenezes you don't think it would be cool to leave the term static site generator? it's something that is consolidated in the community.